codes/codes.go

```diff
@@ -195,6 +195,27 @@ const (
 	_maxCode = 17
 )
 
+// CodeToStr maps from the code type to strings with only uppercase letters.
+var CodeToStr = map[Code]string{
```

Comment:
I'd rather not export this as a map, and make it a permanent part of our API.

Options?  One easy one is putting it in `internal` instead.  (Sadly we decided to output a Go-looking name from `String()`, and changing it to report the canonical name instead would probably break too many tests.)

Maybe a `CanonicalString` method?

Reply:
Please delete this now

Reply:
Done. Whoops forgot to delete sorry.